src/main/Port.Adapter/UI/Views/Blazor.Common/TreeView.razor

```diff
@@ -156,13 +157,15 @@
                             {
                                 <div class="row">
                                     <TreeView Children="@child.Children" 
-                                        AvatarUrl="@this.AvatarUrl"
-                                  @bind-SelectedNeuron="@this.SelectedNeuron" 
-                                        OnMenuRequested="@this.MenuRequested"
-                                        OnInfoRequested="@this.InfoRequested" 
-                                        @bind-ControlsEnabled="@this.ControlsEnabled" 
-                                        IsRoot="false" 
-                                        IsRootRightJustified="@(this.ShouldRightJustifyBasedOnRoot(child))" 
+                                    AvatarUrl="@this.AvatarUrl"
+                                    OnHover="@(async (childNode) => await Hover(childNode))"
+                                    OnUnHover="@(async (childNode) => await UnHover(childNode))"
+                                    @bind-SelectedNeuron="@this.SelectedNeuron" 
+                                    OnMenuRequested="@this.MenuRequested"
+                                    OnInfoRequested="@this.InfoRequested" 
+                                    @bind-ControlsEnabled="@this.ControlsEnabled" 
+                                    IsRoot="false" 
+                                    IsRootRightJustified="@(this.ShouldRightJustifyBasedOnRoot(child))" 
```

Comment:
Restore tab so it is indented away from the TreeView element.

Reply:
Done

Reply:
Please indent lines 154 to 162 so they are one level deeper than line 153.

Reply:
Done